build.gradle

```diff
@@ -174,23 +174,19 @@ ext.artifacts_dir = '-Dartifacts.root=' + artifactsDir
 ext.pluginVersion = "-DversionNumber=" + version
 ext.buildDate = "-DbuildDate=" + new Date().toString()
 ext.extensions_home = '-Dextensions.home=' + rootDir
+ext.skipUnmodifiedModels = "-Dmps.generator.skipUnmodifiedModels=true" //incremental build support
```

Comment:
@coolya So this feature is by default always on and the only way to "deactivate" this is to run `rebuild()`, correct?

Reply:
Yes the rebuild task will also trigger a clean. The clean task  deletes all generated files and MPS will then regenerate them.

Reply:
Its probably easier then to making `-Dmps.generator.skipUnmodifiedModels` configurable from outside.

Reply:
I didn't see much value in configuring it when there is no artefacts MPS shall figure out generating the artefacts. Additionally the clean task comes handy in situations where you just want to get rid of all generated code. For instance when migration. Of course a `git clean -fxd´ might to the same but this was we don't blindly remove all files not in git.